### PR TITLE
Correct misc-misplaced-widening-cast clang warnings

### PR DESCRIPTION
**Pull request by benrubson**
Hi,

This PR helps with #427, it solves clang warnings regarding misc-misplaced-widening-cast, which seems important to care about.

Thank you 👍 

Ben

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -179,7 +179,6 @@ if (${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION} GREATER 3.5) # Need 3.6 or abo
       ",-google-readability-todo"
       ",-google-runtime-int"
       ",-google-runtime-references"
-      ",-misc-misplaced-widening-cast"
```
Comment (rfjakob) [inline CMakeLists.txt]: Why disable the check?

Comment (benrubson) [reply CMakeLists.txt]: It disables the "non-check" :)
It re-enables it 👍

Comment (benrubson) [reply CMakeLists.txt]: All checks are enabled, but the listed ones.

**Comment by benrubson** [comment]
OK for you @rfjakob ?
Let's try to merge these PRs so that we will have a cleaner build log, to easily correct last issues and release 1.9.3 quite soon ? Many thx 👍

**Pull request merged**